src/alto-ui/helpers/__tests__/sum.test.js

```diff
@@ -0,0 +1,24 @@
+import sum from '../sum';
+
+describe('sum', () => {
+  it('is sum', () => {

```

Comment:
```suggestion
  it('should return 0 without params', () => {
```

Reply:
I'm testing by TDD so the first assertion is always default value.
When I wrote that test `sum` function doesn't exists yet.
I think that assertion tells us everything what we want to know in this case `expect(sum()).toBe(0);`